src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -764,18 +782,62 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
       isSearching,
       savedObjectCounts,
     } = this.state;
-    const { http, allowedTypes, applications } = this.props;
+    const { http, allowedTypes, applications, namespaceRegistry } = this.props;
 
     const selectionConfig = {
       onSelectionChange: this.onSelectionChanged,
     };
+    const typeCounts = savedObjectCounts.type || {};
 
     const filterOptions = allowedTypes.map((type) => ({
       value: type,
       name: type,
-      view: `${type} (${savedObjectCounts[type] || 0})`,
+      view: `${type} (${typeCounts[type] || 0})`,
     }));
 
+    const filters = [
+      {
+        type: 'field_value_selection',
+        field: 'type',
+        name: i18n.translate('savedObjectsManagement.objectsTable.table.typeFilterName', {
+          defaultMessage: 'Type',
+        }),
+        multiSelect: 'or',
+        options: filterOptions,
+      },
+      // Add this back in once we have tag support
+      // {
+      //   type: 'field_value_selection',
+      //   field: 'tag',
+      //   name: 'Tags',
+      //   multiSelect: 'or',
+      //   options: [],
+      // },
```

Comment:
If this comments are not needed, could we remove?

Reply:
This comment was already in the code so I chose to leave it as is, but I can remove it.

Reply:
@cwperks Let's not copy the commented code 😉 We have version control, so no need to keep unused code commented.

Reply:
Removed the commented out code